packages/requisitions/src/RequestRequisition/api/api.ts

```diff
@@ -126,6 +126,18 @@ export const getRequestQueries = (sdk: Sdk, storeId: string) => ({
 
       throw new Error('Record not found');
     },
+    lineChartData: async (requisitionLineId: string) => {
+      const result = await sdk.requisitionLineChart({
+        storeId,
+        requisitionLineId,
+      });
+
+      if (result?.requisitionLineChart.__typename === 'ItemChartNode') {
+        return result.requisitionLineChart;
+      }
+
+      throw new Error('Unable to load chart data');
```

Comment:
am swallowing the error when querying, so this is technically redundant.. but neater, I think?